src/main/kotlin/no/nb/mlt/wls/product/service/SynqService.kt

```diff
@@ -0,0 +1,24 @@
+package no.nb.mlt.wls.product.service
+
+import no.nb.mlt.wls.product.payloads.SynqProductPayload
+import org.springframework.beans.factory.annotation.Value
+import org.springframework.http.HttpEntity
+import org.springframework.http.HttpMethod
+import org.springframework.stereotype.Service
+import org.springframework.web.client.RestTemplate
+import java.net.URI
+
+@Service
+class SynqService {
+    val restTemplate: RestTemplate = RestTemplate()
+
+    @Value("\${synq.path.base}")
+    lateinit var baseUrl: String
+
+    fun createProduct(payload: SynqProductPayload) {
+        // TODO - Validation
+        val uri = URI.create("$baseUrl/nbproducts")
```

Comment:
Note to self: check if there are safer ways of creating URIs

Reply:
It seems to be possible to use the UriComponentsBuilder from Spring Framework, however, in our simple case it seems like an overkill for a simple problem
```kotlin
val uri = UriComponentsBuilder
    .fromHttpUrl(baseUrl)
    .path("/nbproducts")
    .build(emptyMap<String, String>())
```